docs/UserGuide.md

```diff
@@ -34,10 +34,10 @@ Let TAManager handles all your TA management needs, while you power up your lect
 
 ## Using this Guide
 
-Whether you're a new user or an experienced one, this guide is designed to help you make the most of our TAManager.
+Whether you're a new user or an experienced one, this guide is designed to help you make the most of TAManager.
 Follow the instructions below based on your familiarity with TAManager.
 
-- **New Users:** If you're just getting started, we recommend to begin with the [Quick Start](#quick-start) section. This will provide you with a step-by-step introduction to TAManager's essential features.
+- **New Users:** If you're just getting started, we recommend to begin with the [Quick Start](#quick-start) section. This will provide you with step-by-step introductions to TAManager's essential features.
```

Comment:
For the last part, I think "step-by-step introductions" sounds weird. Maybe we could say "provide you with a comprehensive introduction to TAManager's essential features"?